examples/docs_beta_snippets/docs_beta_snippets/guides/data-assets/quality-testing/asset-checks/asset-checks-with-schedule-and-sensor.py

```diff
@@ -0,0 +1,63 @@
+import os
+
+import pandas as pd
+
+import dagster as dg
+
+
+@dg.asset
+def orders():
+    orders_df = pd.DataFrame({"order_id": [1, 2], "item_id": [432, 878]})
+    orders_df.to_csv("orders.csv")
+
+
+@dg.asset_check(asset=orders)
+def orders_id_has_no_nulls():
+    orders_df = pd.read_csv("orders.csv")
+    num_null_order_ids = orders_df["order_id"].isna().sum()
+    return dg.AssetCheckResult(
+        passed=bool(num_null_order_ids == 0),
+    )
+
+
+# highlight-start
```

Comment:
@maximearmstrong @erinkcochran87 re: the highlight here, what about just making this one big block of highlighted text?